corehq/apps/builds/jadjar.py

```diff
@@ -1,14 +1,27 @@
-from StringIO import StringIO
 import itertools
-from zipfile import ZipFile, ZIP_DEFLATED
-from django.conf import settings
+import os
 import shlex
+from StringIO import StringIO
 from subprocess import PIPE
 from tempfile import NamedTemporaryFile
-import os
+from zipfile import ZipFile, ZIP_DEFLATED
+
+from django.conf import settings
+from lxml import etree
+
 from dimagi.utils.subprocess_manager import subprocess_context
 
 
+def _make_address_j2me_safe(address):
+    if settings.J2ME_ADDRESS:
+        return address.replace(
+            settings.BASE_ADDRESS, settings.J2ME_ADDRESS, 1
+        ).replace(
+            "https://%s" % settings.J2ME_ADDRESS, "http://%s" % settings.J2ME_ADDRESS
```

Comment:
If this gets deployed to prod before the appropriate redirect code it would force all new builds to use http right?